telemetry/tokens.go

```diff
@@ -0,0 +1,60 @@
+package telemetry
+
+import (
+	cosmostelemetry "github.com/cosmos/cosmos-sdk/telemetry"
+	"github.com/hashicorp/go-metrics"
+)
+
+// MintedTokensFromModule is a function to track token minting from a specific module.
+// The metric used is an increment counter, and the label includes the module name for context.
+func MintedTokensFromModule(module string, amount float32) {
+	if !isTelemetyEnabled() {
+		return
+	}
+
+	// CosmosSDK has a metric called `minted_tokens` (as a part of `mint` module), however it is wrongfully marked a `gauge`.
```

Comment:
Going to open a cosmos-sdk pr to address this

Reply:
@okdas Send the link and #PUC once you have it.

I'll help push it through (for review et al) on their end.